pages/index.js

```diff
@@ -4,45 +4,63 @@ import withRedux from 'next-redux-wrapper';
 
 import CoreLayout from 'components/common/CoreLayout';
 import ArticlesRow from 'components/articles/grid/ArticlesRow';
+import Diary from 'components/articles/Diary';
 import ArticlesComplexRow from 'components/articles/grid/ArticlesComplexRow';
 import Button from 'components/common/Button';
 
-import { ArticleModel } from 'utils/customPropTypes';
+import { ArticleModel, DiaryModel } from 'utils/customPropTypes';
 
 import initStore from 'redux/store';
-import { actions as articlesActions, selectors } from 'redux/ducks/articles';
+import { actions as articlesActions, selectors as articlesSelectors } from 'redux/ducks/articles';
 import { actions as auth } from 'redux/ducks/auth';
 
+import { actions as diaryActions, selectors as diarySelectors } from 'redux/ducks/diary';
 import request from 'utils/request';
 
 import text from 'constants/dictionary';
 
 const mapStateToProps = state => ({
-  articles: selectors.getAll(state),
-  error: selectors.isError(state),
+  articles: articlesSelectors.getAll(state),
+  error: articlesSelectors.isError(state),
+  diary: diarySelectors.getCurrent(state),
+});
+
+const mapDispatchToProps = dispatch => ({
+  getByDay: () => dispatch(diaryActions.getByDay()),
 });
 
 const FIRST_LINE_END = 4;
 
 class HomePage extends Component {
   static propTypes = {
     articles: PropTypes.arrayOf(PropTypes.shape(ArticleModel)).isRequired,
+    diary: PropTypes.shape(DiaryModel).isRequired,
```

Comment:
in #16 I came to the fact that more convenient way is [export](https://github.com/babajka/babajka-frontend/pull/15/files#diff-76e94d767ebb316d865d41e4dc5cfb93) `Shape` & `Array` objects.